scripts/process_image_dir.py

```diff
@@ -5,56 +5,80 @@
 from glob import glob
 from astropy.io import fits
 
+from pocs.utils import current_time
 from pocs.utils.config import load_config
 from pocs.utils.images import clean_observation_dir
 from pocs.utils.google.storage import upload_observation_to_bucket
 from pocs.utils.google.clouddb import add_header_to_db
 
 
-def main(directory, upload=True, send_headers=True, verbose=False):
+def main(directory, upload=True, send_headers=True, verbose=False, **kwargs):
     """Upload images from the given directory. """
 
+    def _print(msg):
+        if verbose:
+            print(msg)
+
     config = load_config()
     try:
         pan_id = config['pan_id']
     except KeyError:
         warn("Can't upload without a valid pan_id in the config")
         return
 
-    def _print(msg):
-        if verbose:
-            print(msg)
-
-    _print("Cleaning observation directory")
-    clean_observation_dir(directory, remove_jpgs=False)
+    _print("Cleaning observation directory: {}".format(directory))
+    try:
+        clean_observation_dir(directory, remove_jpgs=False, verbose=verbose, **kwargs)
```

Comment:
How about adding remove_jpgs to the command line flags, instead of hard-coding it? Could be in another PR.